src/wallet/components/WalletDropdownFundLink.tsx

```diff
@@ -0,0 +1,76 @@
+import { isValidElement, useMemo } from 'react';
+import { FundWalletSvg } from '../../internal/svg/fundWallet';
+import { cn, pressable, text as themeText } from '../../styles/theme';
+import { version } from '../../version';
+import type { WalletDropdownFundLinkReact } from '../types';
+import { getWindowDimensions } from '../utils/getWindowDimensions';
+
+export function WalletDropdownFundLink({
+  className,
+  icon,
+  rel,
+  popupFeatures,
+  openIn = 'tab',
+  target,
+  text = 'Fund wallet',
+  popupSize = 'md',
+}: WalletDropdownFundLinkReact) {
+  const iconSvg = useMemo(() => {
```

Comment:
This logic is identical to the one we have in WalletDropdownLink, we should probably have a shared Hook or Utils for both of them.

Reply:
added